developer_guides/testbench/build_testbench.rst

```diff
@@ -0,0 +1,110 @@
+.. _build_testbench:
+
+Build Testbench and do first run
+################################
+
+This assume the firmware git repository has been cloned and a shell is
+in the top level directory. Run the following scripts to see that the
+build for testbench and test pipeline works correctly.
+
+.. code-block:: bash
+
+   ./scripts/build-tools.sh -t
+   ./scripts/rebuild-testbench.sh
+   ./scripts/host-testbench.sh
+
+The current version of script host-testbench.sh outputs text this if
+the previous steps were successful.
+
+::
+
+   ==========================================================
+   test volume with ./volume_run.sh 16 16 48000 zeros_in.raw volume_out.raw
+   volume test passed!
+   volume_out size check passed!
+   ==========================================================
+   test src with ./src_run.sh 32 32 44100 48000 zeros_in.raw src_out.raw
+   src test passed!
+   src_out size check passed!
+   ==========================================================
+   test eqiir with ./eqiir_run.sh 16 16 48000 zeros_in.raw eqiir_out.raw
+   eqiir test passed!
+   eqiir_out size check passed!
+
+In future there will be more items tested in this quick check step so
+the output format may change. If this worked the testbench can be used
+for audio quality tests and debugging new components under development.
+
+When examining script host-testbench.sh it is seen that e.g. the IIR EQ
+test is run with commands
+
+.. code-block:: bash
+
+   cd tools/test/audio
+   head -c 10240 < /dev/zero > zeros_in.raw
+   ./eqiir_run.sh 16 16 48000 zeros_in.raw eqiir_out.raw
+
+The directory with eqiir_run.sh is entered first. Then a file of 10240
+bytes of zeros is created. As 16 bit data it corresponds to 2560
+frames of S16_LE format stereo frames (4 bytes per frame). At 48 kHz
+rate it corresponds to 5.3 ms of audio stream. Audio test signals are
+usually longer but this was sufficient for quick the testbench health check.
+
+To process e.g. a music file with a under-development SOF component an
+utility to convert from wav, mp3, etc. to raw S16_LE/S24_LE/S32_LE
+format is needed. The next command installs from Ubuntu packages
+repository a bunch of useful tools for audio files converting,
+viewing, recording, playing, and editing. FFMPEG can be used to
+import/export formats that the simpler tools sox and ecasound do not
+support. The last three items are audio light audio waveform viewers
+and players with some editing and mixing capabilities. Alsa digital
+audio workstation (DAW) software can be used such as Ardour, Qtractor,
+and MusE but there's more effort in usim them for small quick tasks
```

Comment:
can be used such as Ardour, Qtractor, and MusE -> such as Ardour, Qtractor,and MusE can be used